config/initializers/seeds.rb

```diff
@@ -1,21 +1,23 @@
-return unless (ForemanInternal.table_exists? rescue(false)) && !Foreman.in_rake? && !Rails.env.test?
+Rails.application.config.to_prepare do
```

Comment:
I believe we want to do this just once, so `after_initialize` would be probably a better fit.